src/azure-cli/azure/cli/command_modules/cosmosdb/custom.py

```diff
@@ -266,6 +266,9 @@ def _create_database_account(client,
     backup_policy = None
     if backup_policy_type is not None:
         if backup_policy_type.lower() == 'periodic':
+            if backup_interval is None and backup_retention is None:
+                raise CLIError(
+                    '--backup-interval and --backup-retention must be specified when Backup Policy Type is Periodic.')
             backup_policy = PeriodicModeBackupPolicy()
             if backup_interval is not None or backup_retention is not None or backup_redundancy is not None:
                 periodic_mode_properties = PeriodicModeProperties(
```

Comment:
Also, updatethe following code:

    elif backup_interval is not None or backup_retention is not None:
        backup_policy = PeriodicModeBackupPolicy()
        periodic_mode_properties = PeriodicModeProperties(
            backup_interval_in_minutes=backup_interval,
            backup_retention_interval_in_hours=backup_retention
        )
        backup_policy.periodic_mode_properties = periodic_mode_properties
        
To:

     elif backup_interval is not None or backup_retention is not None **_or backup_redundancy is not None_**:
          backup_policy = PeriodicModeBackupPolicy()
          periodic_mode_properties = PeriodicModeProperties(
              backup_interval_in_minutes=backup_interval,
              backup_retention_interval_in_hours=backup_retention
          )
          backup_policy.periodic_mode_properties = periodic_mode_properties

Reply:
I am not sure what the ** mean, did you mean to add commented lines?

Reply:
that means add following condition to elif : "or backup_redundancy is not None"

Reply:
Done